### DISCUSSION TITLE
PrimeFaces version updates on blog

### DISCUSSION THREAD

**Discussion by escay** [primefaces/community#40] [PrimeFaces] [answered]
Dear Primefaces,

I often check https://www.primefaces.org/blog/ for updates on PrimeFaces.
For example:
https://www.primefaces.org/primefaces-8-0-23-10-0-18-11-0-11-and-12-0-5-released/
https://www.primefaces.org/primefaces-12-0-4-released/
https://www.primefaces.org/primefaces-11-0-10-and-12-0-3-released/
are very useful.

But since version 13, it does not seem to get updates on the site anymore?

I did find this release 13.0.0 release message by accident:
https://dev.to/cagataycivici/primefaces-v1300-released-6io

But looking in maven I see 13.0.1 has been released a few days ago:
https://search.maven.org/artifact/org.primefaces/primefaces/13.0.1/jar

Will the https://www.primefaces.org/blog/ be used again in the future or is there another source to keep track of new versions?

Best regards,
Martijn

**Comment by jepsar** [comment]
I don't know about the blog, but if you want to be notified of (community) releases of PrimeFaces, simply watch the PrimeFaces repository. You might want to set it to custom:

<img width="535" alt="Screenshot 2023-09-14 at 09 14 45" src="https://github.com/primefaces/community/assets/7500178/ab0517b6-ea76-49af-af46-045d6499e200">

**Comment by onursenture** [comment, marked as answer]
Hello @escay,

We continue to use the blog. However, there has been a miscommunication in our workflow as we've now also started publishing on Dev.to. I've added the missing [PrimeFaces 13.0.0 announcement](https://www.primefaces.org/primefaces-v13-0-0-released/) to the blog. We haven't written an announcement for 13.0.1 yet, but that will also be published on the blog.

As @jespar also mentioned, the most efficient recommended way to follow is through [GitHub Releases](https://github.com/primefaces/primefaces/releases), but we make necessary announcements through channels like the Blog, [newsletter](https://www.primefaces.org/newsletter), pinned topics in GitHub discussions, [Discord](https://discord.gg/gzKFYnpmCY), and [Twitter](https://twitter.com/primefaces).

P.S: https://www.primefaces.org/primefaces-8-0-23-10-0-18-11-0-11-and-12-0-5-released/ This is the latest announcement and update concerning the mentioned versions.

Apart from that, we've also added the missing links we noticed on the https://www.primefaces.org/downloads/ page thanks to your message.

Thank you for your feedback. 🙏
